src/main/java/frc/robot/commands/RunBelts.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+
+/** Command that runs belt motors when started, and stops belt motors when ended. */
+public class RunBelts extends Command {
+  /** Creates a new RunBelts. */
+  public RunBelts() {
+    // Use addRequirements() here to declare subsystem dependencies.
```

Comment:
Adapt this to use its own copy of MailboxBelts and remember to call addRequirements()